src/main/java/svattask/demo/application/PerformanceData.java

```diff
@@ -0,0 +1,11 @@
+package svattask.demo.application;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+public class PerformanceData {
+
+    public final static List<List<String>> COLLECTION_TYPES = List.of(new ArrayList<>(), new LinkedList<>());
```

Comment:
1. I don't see much sense in this method of implementation (creating a new data object for the collection constant), but okay.
2. should be - public static final